### PR TITLE
[CDAP-8019] Introduce a configuration for the Explore HTTP timeout

### PR DESCRIPTION
**Pull request by anew**
subject says it all

### CODE REVIEW

**Comment by anew** [comment]
@awholegunch I added the Docs tag to make sure you take a look at the description for the new property in cdap-defaults.xml.

**Comment by bdmogal** [comment]
@anew code LGTM and will fix the issue I reported. You had a comment to have the timeout be passed on a call by call basis on the jira... Did we decide to not go that route? I see that the description has a guard though.

Also, whenever @awholegunch is ok with the description, you'll have to update the MD5 of cdap-default.xml.

**Comment by anew** [comment]
Oh I meant my second comment to override the first, because it addresses the concern that caused the first one. Call-by-call basis would be really hard (it would change all the APIs to accepts a new parameter) and also, who would use it where? So I decided to go with the simpler approach. Also, if we have a need for it in the future, we can still think about doing it on a call-by-call basis.

File: cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java [hunk 1]
```diff
@@ -54,6 +55,7 @@ public ExploreFacade(ExploreClient exploreClient, CConfiguration cConf) {
     if (!exploreEnabled) {
       LOG.warn("Explore functionality for datasets is disabled. All calls to enable explore will be no-ops");
```
Comment (awholegunch) [inline cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java]: Shouldn't this end with a period?

**Comment by awholegunch** [comment]
I put the properties in alphabetical order, and changed the text to make them consistent. When we refer to another property in a description, we put the other property as the `${other.property}`.

**Comment by anew** [comment]
Build passed here: http://builds.cask.co/browse/CDAP-DUT5442/latest.

Will squash and merge now.

**Pull request closed without merging**